Doc/c-api/tuple.rst

```diff
@@ -35,6 +35,10 @@ Tuple Objects
 
    Return a new tuple object of size *len*, or *NULL* on failure.
 
+   .. versionchanged:: 3.7
+
+      ``PyTuple_New(0)`` always succeeds and never returns *NULL*.
```

Comment:
We don't need, but I consider this as an enhancement. Yes, this promise adds limitations on future changes, but I afraid that if don't document this property explicitly, we can forgot about this and unintentionally change it. However changing this makes using of `_PyTuple_Empty` unsafe.